index.html

```diff
@@ -216,13 +218,13 @@ <h3>Profesional Art Printing Data</h3>
     <h2 class="about_me_title">About me.</h2>
     <p class="about_me_content">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 
```

Comment:
- [ ] Nice work here, but your background in the "about-me" section is overflowing 🔽 which causes the appearance of a horizontal scrollbar, please fix this.

![2023-04-01_000203](https://user-images.githubusercontent.com/104319462/229254720-c510b015-99a3-4336-a88e-bf8dad5afe89.png)

- [ ] Please make sure to also reduce the top and bottom padding in the "about-me" and "contact" sections, because they are oversized and don't match the Figma template.